code/modules/client/preferences.dm

```diff
@@ -945,12 +951,32 @@ var/const/MAX_SAVE_SLOTS = 10
 				if("pred_boot_type")
 					var/new_predator_boot_type = input(user, "Choose your greaves type:\n(1-4)", "Greave Selection") as num|null
 					if(new_predator_boot_type) predator_boot_type = round(text2num(new_predator_boot_type))
+				if("pred_mask_mat")
+					var/list/options = list("ebony", "silver", "bronze")
+					var/new_pred_mask_mat = tgui_input_list(user, "Choose your mask material:", "Mask Material", options)
+					if(!new_pred_mask_mat)
+						return
+					predator_mask_material = new_pred_mask_mat
 				if("pred_armor_mat")
 					var/list/options = list("ebony", "silver", "bronze")
 					var/new_pred_armor_mat = tgui_input_list(user, "Choose your armour material:", "Armor Material", options)
 					if(!new_pred_armor_mat)
 						return
 					predator_armor_material = new_pred_armor_mat
+				if("pred_greave_mat")
+					var/list/options = list("ebony", "silver", "bronze")
+					var/new_pred_greave_mat = tgui_input_list(user, "Choose your greave material:", "Greave Material", options)
+					if(!new_pred_greave_mat)
+						return
+					predator_greave_material = new_pred_greave_mat
+				if("pred_flavor_text")
+					var/pred_flv_raw = input(user, "Choose your Predator's flavor text:", "Flavor Text", predator_flavor_text) as text|null
+					if(pred_flv_raw)
+						var/predator_flv_txt = reject_bad_text(pred_flv_raw, 40)
```

Comment:
Use a string size define instead of a hard number (just copy regular flavour texts length) and implement a cool down on saving it too if you make it significantly bigger